src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -930,8 +930,8 @@ export default function ({
     }
 
     if (
-      lastEvaluatedToken.position.column !== currentToken.position.column ||
-      lastEvaluatedToken.position.lineNumber !== currentToken.position.lineNumber ||
+      (lastEvaluatedToken.position.column !== currentToken.position.column ||
+        lastEvaluatedToken.position.lineNumber !== currentToken.position.lineNumber) &&
```

Comment:
nit: Could you please add a comment here to explain what the boolean expressions do, just to make the code a little easier to understand?